docs/docs/jsx-in-depth.md

```diff
@@ -245,6 +245,32 @@ function App2() {
 
 Spread attributes can be useful when you are building generic containers. However, they can also make your code messy by making it easy to pass a lot of irrelevant props to components that don't care about them. We recommend that you use this syntax sparingly.
 
+### ES6 Object Shorthand
+
+If you want to filter the props that you want to pass, you can do it by using ES6 Object Shorthand in combination with object spread operator and object destructuring assignment:
+
+```js{4}
+render(){
+  const {myPropsOne, myPropsTwo} = this.props;
+  return (
+    <MyComponent {...{myPropsOne, myPropsTwo}} />
```

Comment:
Please use more realistic prop names than `myPropsOne`, `myPropsTwo`. Maybe `onClick` and `value`.